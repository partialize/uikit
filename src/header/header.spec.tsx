import React from 'react';
import renderer from 'react-test-renderer';
import Header from "./header";

test('my pretty Container', () => {
  const tree = renderer.create(
    <Header/>,
  );

  expect(tree).toMatchSnapshot();
});
