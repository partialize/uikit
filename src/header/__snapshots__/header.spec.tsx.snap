// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my pretty Container 1`] = `
<h1
  className="h2f324r"
>
  Hello world
</h1>
`;
